public/index.html

```diff
@@ -2,14 +2,17 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Mozfest 2015</title>
-  <link rel="stylesheet" href="index.css"/>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:800,600,400,300" rel="stylesheet" type="text/css">
-  <link href="https://mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" rel="stylesheet" />
-  <link rel="stylesheet" type="text/css" href='https://css.tito.io/v1' />
-  <link rel="stylesheet" type="text/css" href='https://api.tiles.mapbox.com/mapbox.js/v2.1.5/mapbox.css' />
-  <link rel="stylesheet" href="build/bundle.css">
+  <title>Mozfest 2015</title>
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png">
+  <link rel="icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:800,600,400,300">
+  <link rel="stylesheet" type="text/css" href="https://mozorg.cdn.mozilla.net/media/css/tabzilla-min.css">
+  <link rel="stylesheet" type="text/css" href="index.css">
```

Comment:
ummm I'm not sure if loading `index.css` before `https://css.tito.io/v1`, `mapbox.css`, and `build/bundle.css` is intended. Let's just keep it as is for now